superset/models.py

```diff
@@ -1149,6 +1149,9 @@ def visit_column(element, compiler, **kw):
             qry.compile(
                 engine, compile_kwargs={"literal_binds": True},),
         )
+        template_processor = get_template_processor(
```

Comment:
Sorry,i remove it.this is for jinja {{}} in the SQL ,no only in WHEWE or HAVING.